packages/pluggableWidgets/datagrid-dropdown-filter-web/src/components/__tests__/FilterComponent.spec.tsx

```diff
@@ -175,4 +180,97 @@ describe("Filter selector", () => {
             });
         });
     });
+
+    describe("focus", () => {
+        beforeEach(() => (document.body.innerHTML = ""));
```

Comment:
Why do u exactly need this?

Reply:
jestdom keep the dom unless you explicitly clear it. I discovered this due to the fact that i was getting old DOM from previous tests. Maybe this also needs to be added for TreeView.

Reply:
https://stackoverflow.com/questions/42805128/does-jest-reset-the-jsdom-document-after-every-suite-or-test

Reply:
Hmm weird, I didn't come across this. Will take a look for tree view